code/src/components/todosummary.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { todos } from '../reducers/todos.js';
+
+export const Summary = () => {
+  // pick up the list from the store
+  const list = useSelector((store) => store.todos.list);
```

Comment:
Since you're only interested in the items-array - it's better to get only that:
`const items = useSelector((store) => store.todos.list.items);`